src/main/webapp/app/core/user/account.model.ts

```diff
@@ -4,7 +4,7 @@ export class Account {
     public activated?: boolean;
     public authorities?: string[];
     public login?: string;
-    public email?: string;
+    public visibleEmail?: string;
```

Comment:
Same here. I don't think it makes all that much sense to call it `visibleEmail` on the client.